### PR TITLE
minor cleanups

### PR DESCRIPTION
**Pull request by dfa1**
- redundant suppressions
- unused imports
- final field

### CODE REVIEW

File: src/main/java/net/jodah/failsafe/util/concurrent/Scheduler.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
    * The default scheduler used by Failsafe if no other scheduler or {@link ScheduledExecutorService} is configured for
    * an execution.
    */
-  public static Scheduler DEFAULT = DelegatingScheduler.INSTANCE;
```
Comment (jhalterman) [inline src/main/java/net/jodah/failsafe/util/concurrent/Scheduler.java:34]: This field is meant to be part of the API which is why it's public static.

Comment (dfa1) [reply src/main/java/net/jodah/failsafe/util/concurrent/Scheduler.java:34]: can't remember why I removed this... looks wrong now, thank you :)

Comment (dfa1) [reply src/main/java/net/jodah/failsafe/util/concurrent/Scheduler.java:34]: ah yes, context is that `Scheduler` is an interface and by default all fields are `public static`:

> Every field declaration in the body of an interface delaration is implicitly public, static, and final. It is permitted to redundantly specify any or all of these modifiers for such fields.

https://docs.oracle.com/javase/specs/jls/se16/html/jls-9.html#jls-9.3

Comment (jhalterman) [reply src/main/java/net/jodah/failsafe/util/concurrent/Scheduler.java:34]: Ah good catch 👍

**Pull request merged**